flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/TwoInputStreamTaskTest.java

```diff
@@ -51,14 +53,14 @@
 import org.apache.flink.streaming.util.TestBoundedTwoInputOperator;
 import org.apache.flink.streaming.util.TestHarnessUtil;
 
+import org.hamcrest.collection.IsMapContaining;
 import org.junit.Assert;
 import org.junit.Test;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
+import java.util.*;
```

Comment:
nit: I think we do not allow wildcard[ imports](https://flink.apache.org/contributing/code-style-and-quality-formatting.html#imports)

Reply:
Yes, my home mac setting was wrong, did it automatically. I will fix it.